MapboxCoreNavigation/RouteController.swift

```diff
@@ -393,4 +464,20 @@ extension RouteController: CLLocationManagerDelegate {
         
         incrementRouteProgress(alertLevel, location: location, updateStepIndex: updateStepIndex)
     }
+    
+    func convertLocastionsObject(locations: [CLLocation]) -> [[String: Any]] {
+        return locations.map { (location: CLLocation) -> [String : Any] in
+            var locationDictionary:[String: Any] = [:]
+            locationDictionary["lat"] = location.coordinate.latitude
+            locationDictionary["lng"] = location.coordinate.latitude
+            locationDictionary["altitude"] = location.altitude
+            locationDictionary["timestamp"] = location.timestamp
```

Comment:
For some reason, I'm unable to todo `location.timestamp.ISO8601` here even though `location.timestamp` is of type `Date`